broadcast/broadcast-client-mock/mock_client_builder.go

```diff
@@ -40,6 +41,8 @@ func (cb *builder) WithMockArc(mockType MockType) *builder {
 		clientToReturn = mocks.NewArcClientMockFailure()
 	case MockTimeout:
 		clientToReturn = mocks.NewArcClientMockTimeout()
+	case MockNillQueryTransactionResponse:
```

Comment:
Please make it a bit shorter, for example:
`MockNilQueryTxResp`

Reply:
Changed to shorter name